aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -1758,8 +1758,17 @@ namespace at::native {
       std::optional<std::string_view> algorithm) {
 #if AT_MKLDNN_ENABLED()
 
-    if (act.dim() == 3 || act.dim() == 5) {
-      // Conv1D/3D post op check
+    if (act.dim() == 3) {
+      // Conv1D post op check
+      TORCH_CHECK(
+        attr == "none" || attr == "relu",
+        "quantized pointwise conv",
+        act.dim()-2,
+        "d doesn't support unary_post_op fusion. Got unary_post_op: ",
+        attr,
+        ".")
+    } else if (act.dim() == 5) {
```

Comment:
Can we merge the checks for different dims to save code?

Reply:
Different dims have different checks so hard to merge. And they are not merged before my changes. Thanks.

Reply:
I have refactored this part to save code. Thanks.